src/main/java/frc/robot/subsystems/OldClimber.java

```diff
@@ -178,11 +153,21 @@ public void stop() {
         setMotorSpeed(0, 0);
     }
 
-    /*
-     * TODO :
-     * - ask about motors
-     * - ask about sensors
-     */
+    public double getLeftArmPos() {
+        return leftWinchMotor.getSelectedSensorPosition() * ENCODER_TICKS_TO_ARMS_LENGTH;
+    }
+
+    //1.65 1.25 - 26.5 in (what is this?)
+    public double getRightArmPos() {
```

Comment:
Currently, 0.0 - 1.0, is the percentage of the climber's arm. Could be in inches or an `armState` enum. Definitely needs JavaDocs